### PR TITLE
[FIX] Fix/schedulling validation onsubmit

### PR DESCRIPTION
**Pull request by coderwithattitude**

### CODE REVIEW

File: app/components/ScheduleWizard/ConfirmSettings.js [hunk 1]
```diff
@@ -58,6 +78,9 @@ class ConfirmSettings extends Component {
         <div className="row">
 
           <div className="col-sm-6 col-md-6">
+            {this.infoSettingsValidations()}
+            {this.bountySettingsValidations()}
+            {((scheduleStore.isUsingTime && this.timeSettingsValidations()) || this.blockComponentValidations())}
```
Comment (Bagaric) [inline app/components/ScheduleWizard/ConfirmSettings.js]: These three lines not be posted under `col-md-6` but above `<div className="row">`.

Comment (coderwithattitude) [reply app/components/ScheduleWizard/ConfirmSettings.js]: @Bagaric thanks for the feedback, i have updated accordingly

Comment (coderwithattitude) [reply app/components/ScheduleWizard/ConfirmSettings.js]: @Bagaric kindly update your review

**Comment by adibas03** [comment]
Merging now

**Pull request merged**